CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.52.3      2023-11-30
+
+* Reduce frequency of collector log generation
+  - Previously, real-time snapshots would generate log lines every 10 seconds,
```

Comment:
I realize this contradicts @seanlinsley's feedback, but I'm not a fan of introducing a new term here that is different from what we document (e.g. on https://pganalyze.com/docs/collector#data-collection-and-snapshot-submission-frequency).

Maybe a compromise for now could be to use the term "near real-time" (which we do use in-app on some check pages), and specifically reference the "compact" term:

```suggestion
  - Previously, near real-time "compact" snapshots would generate log lines every 10 seconds,
```

Reply:
Nice, I think it's good combination of near real-time + compact (and I was mildly panicking that the doc is not having updated format of the log while I was 100% sure Maciek already fixed it, but I just realized that we just haven't shipped it yet).